### PR TITLE
new json structure changes

### PR DESCRIPTION
**Pull request by hasancelik**

### CODE REVIEW

File: docs-content/using.html.md.erb [hunk 1]
```diff
@@ -31,6 +31,10 @@ hazelcast       t2.micro, t2.small, m3.medium    Hazelcast Service
 
 1. Create a file named `hazelcast.json` using this [sample Hazelcast JSON file](https://github.com/hazelcast/hazelcast-code-samples/blob/master/hazelcast-integration/pcf-integration/hazelcast.json) as a template.
 
+<p class="note"><strong> Note:</strong>
+ Hazelcast IMDG Enterprise for Pivotal Platform v2.0.0
+ and later require JSON config file that uses new structure. For more information about JSON config Structurr, see <a href="./json-config">JSON Config Structure</a> </p> section.
```
Comment (googlielmo) [inline docs-content/using.html.md.erb]: Typo: "Structurr" => "structure"

**Review by googlielmo — APPROVED** [review]
One typo to be corrected, other than that LGTM

File: docs-content/using.html.md.erb (hunk 1 above)
Comment (hasancelik) [reply docs-content/using.html.md.erb]: fixed and rebased 👍

**Comment by AndrewCovarruPivotal** [comment]
@hasancelik Merged PR. Thanks!

**Pull request merged**